webview/src/experiments/components/table/CellRowActions.tsx

```diff
@@ -45,26 +40,16 @@ export const CellRowAction: React.FC<CellRowActionProps> = ({
   const count = (showSubRowStates && subRowsAffected) || 0
 
   return (
-    <Tooltip
-      content={tooltipContent}
-      placement="bottom-end"
-      delay={[CELL_TOOLTIP_DELAY, 0]}
-    >
+    <CellHintTooltip tooltipContent={tooltipContent}>
       <div
         className={cx(styles.rowActions, hidden && styles.hidden)}
         data-testid={testId}
       >
-        <Indicator
-          count={count}
-          tooltipContent={
-            count &&
-            `${count} ${pluralize('sub-row', count)} ${actionAppliedLabel}.`
-          }
-        >
+        <Indicator tooltipContent="" count={count}>
```

Comment:
Originally, each indicator has a tooltip but that caused some overlap:

<img width="148" alt="image" src="https://user-images.githubusercontent.com/43496356/189775616-6d1c3254-e3c0-4c2a-9c8d-d946e4f26846.png">

I just took it off for now, but we could also just keep it as is or have a single tooltip that gives information on action and sub items.

Reply:
Maybe `tooltipContent` should be optional in `<Indicator />` and the markup should change whether the prop is passed or not.